.github/workflows/dotnet.yml

```diff
@@ -17,6 +17,7 @@ jobs:
   build:
 
     runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
```

Comment:
Had a couple of builds that went stuck for hours. 

![image](https://github.com/user-attachments/assets/4d53f152-2150-4361-8a1d-5de07ce56c0f)


At first, I thought it was a concurrency related issues, but it wasn't eventually the case. Maybe a transient GitHub actions issue?

Whatever, let's try to bail out early when that happens and avoid wasting those precious free build minutes.

Reply:
I like that!